### ISSUE TITLE
Example

### ISSUE THREAD

**Issue by wahmedswl** [takama/daemon#37]
Hi, 
   Could you plz provide simple example where actual logic should be implemented. In the example given, if its been called with arguments eg 
```
myservice install
```
will the later portion be called? It would be great if simple example would be provided that would allow all actions like install/start/stop etc just like in the provided example but simplified.

Thanks

**Comment by derkan** [comment]
Extending example at README:
```go
....
	switch command {
	case "install":
		deleted := 0
		for i := range args {
		    j := i - deleted
		    if args[j] == "install" {
			args = args[:j+copy(args[j:], args[j+1:])]
			deleted++
		    }
		}
		return service.Install(args...)
....
```
And install service with:
```bash
myservice install -config x.ini
```

**Comment by maximus12793** [comment]
yes can we add this to readme? It is not clear how install/start etc. all work when the only example is using an eventloop. I am trying to use this for a cron job and would be nice to have some more examples
